new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSimpleNginxValidation.java

```diff
@@ -628,4 +673,149 @@ private void checkServiceCreated(String serviceName) {
+                podName,
+                domainNamespace,
+                condition.getElapsedTimeInMS(),
+                condition.getRemainingTimeInMS()))
+        .until(assertDoesNotThrow(() -> podDoesNotExist(podName, null, domainNamespace),
+            String.format("Pod %s still exists in namespace %s", podName, domainNamespace)));
+  }
+
+  /** Scale the WebLogic cluster of the domain to numberOfServers in the specified domain namespace.
+   *  Verify the pods are created or deleted depending on the numberOfServers. Also verify
+   *  NGINX can access the sample apps to all the servers after the scale operation.
+   *
+   * @param domainUid the domain with domainUid which will be scaled
+   * @param domainNamespace the domain namespace the domain resides
+   * @param clusterName the WebLogic cluster name in the domain to be scaled
+   * @param replicasBeforeScale the replicas of the WebLogic cluster before the scale
+   * @param numberOfServers the number of servers to be scaled to
+   * @return the replicas of the WebLogic cluster after the scale
+   */
+  private int scaleAndVerifyDomain(String domainUid,
+                                   String domainNamespace,
+                                   String clusterName,
+                                   int replicasBeforeScale,
+                                   int numberOfServers) {
+
+    String manageServerPodNamePrefix = domainUid + "-" + clusterName + "-managed-server";
+
+    // scale the cluster in the domain
+    assertThat(assertDoesNotThrow(() -> scaleDomain(domainUid, domainNamespace, clusterName, numberOfServers)))
+        .as(String.format("Verify scale the cluster %s of domain %s in namespace %s",
+            clusterName, domainUid, domainNamespace))
+        .withFailMessage(String.format("can not scale cluster %s of domain %s in namespace %s",
+            clusterName, domainUid, domainNamespace))
+        .isTrue();
+
+    // generate a curl command to ping the sample app through the ingress controller
+    curlCmd = generateCurlCmd(clusterName);
+
+    // generate the expected server list which should be in the sample app response string
+    List<String> expectedServerNamesInAppResponse =
+        generateExpectedServerNamesInAppResponse(clusterName, replicasBeforeScale);
+    logger.info("expected server name list which should be in the sample app response: {0} before scale",
+        expectedServerNamesInAppResponse);
+
+    if (replicasBeforeScale <= numberOfServers) {
+
+      // check that NGINX can access the sample apps from the original managed servers in the domain
+      logger.info("Check that NGINX can access the sample app from the original managed servers in the domain "
+                  + "while the domain is scaling up.");
+      assertThat(callWebAppAndCheckForServerNameInResponse(curlCmd, expectedServerNamesInAppResponse, 50))
+          .as("Verify NGINX can access the sample app from the original managed servers in the domain")
```

Comment:
Verify
```suggestion
          .as("NGINX can access the sample app from the original managed servers in the domain")
```

Reply:
.as() message is not for a successful assertion.